sdk/eventhub/event-hubs/consuming-events.md

```diff
@@ -0,0 +1,85 @@
+# Consuming events
+
+The `EventHubConsumerClient` provides a _push-based_ model for receiving events.
```

Comment:
I wonder if this information would make more sense within the context of our samples, rather than as a separate document?

Reply:
I'm not opposed to interspersing this doc in a sample. I thought this might be more visible but can see the benefit to having this alongside a fully-working sample.

@ramya-rao-a  thoughts on converting this doc to a sample vs keeping this as separate doc?

Reply:
The reference docs on the `SubscriptionEventHandlers` would be the right place for these.

Reply:
I can help with that as my last PR did half of the above work in reference docs already

Reply:
https://github.com/Azure/azure-sdk-for-js/pull/6791/commits/f9fd7d02dc62ac9c0c1041e8b012b19a754581cd